packages/world/src/interfaces/IDelegationControl.sol

```diff
@@ -1,6 +1,16 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.0;
 
-interface IDelegationControl {
+import { IWorldContextConsumer } from "./IWorldContextConsumer.sol";
+import { ERC165 } from "./ERC165.sol";
+
+// ERC-165 Interface ID (see https://eips.ethereum.org/EIPS/eip-165)
+bytes4 constant DELEGATION_CONTROL_INTERFACE_ID = IDelegationControl.verify.selector ^
+  IWorldContextConsumer._msgSender.selector ^
+  IWorldContextConsumer._msgValue.selector ^
+  IWorldContextConsumer._world.selector ^
+  ERC165.supportsInterface.selector;
```

Comment:
is it necessary to define the inherited selectors as well as our own?

and if so would it be easier to do the whole interface ID instead of each selector (caveat: I don't know what the bitshifting is doing here)?

```solidity
bytes4 constant DELEGATION_CONTROL_INTERFACE_ID = IDelegationControl.verify.selector ^
  WORLD_CONTEXT_CONSUMER_INTERFACE_ID ^
  ERC165_INTERFACE_ID;
```

Reply:
agree this is cleaner, and should work since `^` (bitwise XOR) is commutative and associative